src/sidepanel.hpp

```diff
@@ -6,6 +6,8 @@
 
 #include <QTabWidget>
 
+class SearchView;
```

Comment:
The headers are all over the place, with way too many circular dependencies, mostly with `MainWindow`, which really needs to be cleaned up, I just haven't gotten around to doing it yet, so this is fine.